modules/manage/pages/security/authorization/rbac.adoc

```diff
@@ -95,6 +99,24 @@ rpk security role create red
 Successfully created role "red"
 ----
 
+Redpanda Console::
+
+Administrators use Redpanda Console to create a new role by following these steps:
+
+1. Select the `Access Control` tab then select the `Roles` tab.
+
+2. Click the Create Role button.
+
+3. In the `Create Role` view, provide a name for the role and an optional origin host for users to connect from.
+
+4. Define the permissions (ACLs) for the role. You may create ACLs for topics, consumer groups, and transactional IDs. You may also restrict your new role to a specific cluster.
```

Comment:
>You may also restrict your new role to a specific cluster

Is this a console-specific abstraction? within redpanda, there's no way to directly restrict a role to a specific cluster; instead, you would need to bind each ACL to the desired cluster. Maybe @bojand can comment?

Reply:
This was a misinterpretation of the screenshots when I was initially documenting it. I updated this in another place, I'll fix it here! It should read now that, "You may also set permissions across the entire cluster."